### PR TITLE
Updating eclipse software files

### PR DESCRIPTION
**Pull request by DT3264**
Updating eclipse software files

This PR
* Updates eclipse to download the tar file from the eclipseide site instead of a local file in the software folder

### CODE REVIEW

**Review by equetzal — APPROVED** [review]
LGTM!

**Comment by DT3264** [comment]
@equetzal I updated the script to avoid any need for ui interaction as seen on the ioi image, so i think a re-review might be worth,
I already tested this version and works just fine.

File: software-modules/base/03-budgie/files/autostart/autosetup.desktop
Comment (equetzal) [inline software-modules/base/03-budgie/files/autostart/autosetup.desktop:1]: How is this file related to eclipse? 
If it's not related, please delete it from the PR. You uncommit it, delete the files, the commit it again and then link it with `sl pr link 170` (with the new commit checked out)
You can also use sl split but I've not used it.

File: software-modules/programming/eclipse/eclipse.sh
Comment (equetzal) [inline software-modules/programming/eclipse/eclipse.sh:1]: Amazing work on this!, This was something really needed!
One step more to automate builds!

**Review by equetzal — CHANGES_REQUESTED** [review]
Amazing PR, Good work on this! :) 
Let's keep a look into the IOI vm code so that we can learn a bit more how are they doing things.

Requesting changes as the files are on the PR not related to the PR. After remove, AC.

File: software-modules/base/03-budgie/files/autostart/autosetup.desktop
Comment (DT3264) [reply software-modules/base/03-budgie/files/autostart/autosetup.desktop:1]: Whops, i don't know how those files ended up in that PR :sweat_smile: 
Deleting those

**Review by equetzal — APPROVED** [review]

**Pull request merged**